src/components/reactModal.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
```

Comment:
1. You can rename this modal to something that's more related to our project, e.g. `TaskTrackerModal`. Since ReactModal feels like it is exported from `react-modal`.
2. You should render Modal from this component rather than rendering it from `whatsNewModal.tsx`. Reason is that this component should act as the default Modal for our application. 
3. This component should create the barebone UI structure of the Modal.

Reply:
Please make this an HOC. For reference you can look into the `PageScreen.tsx`